src/utils/code.ts

```diff
@@ -48,13 +48,17 @@ const buildParams = (paramsName: any) => {
   paramsName.forEach((param: any) => {
     const optional = param.optional ? '?' : ''
     paramTypes += `${param.name}${optional}: ${param.type}, `
-    let operand =
-      param.type == 'string'
-        ? `'${param.value}'`
-        : param.type == 'Function'
-        ? `${param.value.slice(1, param.value.length - 1)}`
-        : `${param.value}`
-    params += `${param.name}=${operand}, `
+    if (ocTsx) {
```

Comment:
you are adding default value when ocTsx is true. Which means the file inside bit component right?
But we wanted the opposite

Reply:
No, there are two functions - generateCode which generates the OcTsx in bit component and generateOcTsx which generates OcTsx in custom-components.

Reply:
Improved naming and fixed minor error in latest commit